composer.json

```diff
@@ -22,7 +22,7 @@
         }
     },
     "require": {
-        "php": ">=5.3"
+        "php": ">=5.4"
```

Comment:
I understand that as you can see in comment. You may consider to use that changeset in new major version or some other branch.
I just wanted to let you know that json problem exists and there is simple solution